pkg/proc/gdbserial/rr.go

```diff
@@ -15,6 +15,11 @@ import (
 	"github.com/go-delve/delve/pkg/proc"
 )
 
+const (
+	delveRecordFlagsEnvVar = "DELVE_RECORD_FLAGS"
```

Comment:
I think we should name these `DELVE_RR_*` just in case we ever happen to switch to another record & replay backend which doesn't accept the given flags, and just to make things more explicit.

Reply:
Done.